firewood/src/db.rs

```diff
@@ -375,7 +375,8 @@ impl<S: ShaleStore<Node> + Send + Sync> DbRev<S> {
         values: Vec<V>,
     ) -> Result<bool, ProofError> {
         let hash: [u8; 32] = *self.kv_root_hash()?;
-        let valid = proof.verify_range_proof(hash, first_key, last_key, keys, values)?;
+        let valid =
+            proof.verify_range_proof::<K, V, Bincode>(hash, first_key, last_key, keys, values)?;
```

Comment:
```suggestion
            proof.verify_range_proof::<_, _, Bincode>(hash, first_key, last_key, keys, values)?;
```
It's a best practice to let the compiler infer as much as possible to minimize the diff in case you decide to change anything.